core/ecschema-rpc/common/src/SchemaRpcInterface.ts

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { IModelRpcProps, RpcInterface, RpcManager } from "@bentley/imodeljs-common";
+import { SchemaProps } from "@bentley/ecschema-metadata";
+
+/***
+ * Defines an RPC interface to get schema information from a given iModel context.
+ * Method @see getSchemaNames will return the names of schemas that live in this iModel.
+ * The actual schemas can be downloaded using @see getSchemaJSON to get the schema as JSON props
+ * or @see getSchemaXml to get the schemas as XML document.
```

Comment:
Release tag? Usually rpc interfaces are `internal` and a public API that uses them is provided.